config/environments/development.rb

```diff
@@ -58,6 +58,10 @@
   # Raises error for missing translations
   # config.action_view.raise_on_missing_translations = true
 
+  # Enable locale fallbacks for I18n (makes lookups for any locale fall back to
+  # the I18n.default_locale when a translation cannot be found).
+  # config.i18n.fallbacks = [I18n.default_locale]
```

Comment:
Adding these as commented-out options because I spent a second scratching my head over why I was getting this behavior in dev 😅
<img width="1183" alt="fallbacks disabled; 'translation missing'" src="https://user-images.githubusercontent.com/5402927/68414096-6610c380-0144-11ea-9fd0-bc089155b87b.png">